src/status_im/transport/message/core.cljs

```diff
@@ -49,7 +49,7 @@
         ^js visibility-status-updates  (.-statusUpdates response-js)
         ^js current-visibility-status  (.-currentStatus response-js)
         ^js bookmarks                  (.-bookmarks response-js)
-        ^js settings                   (.-settings response-js)
+        ^js settings                   (remove nil? (.-settings response-js))
```

Comment:
I fully agree, there's an open issue for the error https://github.com/status-im/status-mobile/issues/14891#issuecomment-1406257530 and @alwx is taking a look at it, but the fix should be in status-go.

For this PR, I wanted to just prevent the crash, since I have been doing already my fair share of yak shaving to get this PR through :), but this should be reverted and fixed in status-go, I agree.